website/src/components/views/CronJob.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import JobStatusIcon from '../JobStatusIcon.jsx';
+import NavBar from '../NavBar.jsx';
+
+class CronJob extends React.Component {
+    render() {
+        return (
+            <React.Fragment>
+                <NavBar {...this.props} />
+                <CronJobTable {...this.props}></CronJobTable>
+            </React.Fragment>
+        );
+    }
+}
+
+export default CronJob
```

Comment:
Prefer `named` exports rather than `default` exports i.e.

```jsx
export { CronJob }
```

You then import them in the same way i.e.

```jsx
import { CronJob } from '/path/to-CronJob/file'
```